docker_entrypoint.sh

```diff
@@ -1,6 +1,5 @@
 #!/bin/bash
-
-set -ex
+set -e
```

Comment:
Considering allowing for `-x` when needed:

```bash
if [ "${DEBUG:-}" ]; then;
  set -x                                                                                                     
fi
```

Reply:
I don't think we need the debug mode for the entrypoint script at all. It was/is only useful when working on that specific script. That almost never changes at the moment and during development, one can add the flag.

Re general log levels, there is a `--log-level` flag on the `bootstrap` command already, which should be sufficient.